lib/View.js

```diff
@@ -125,7 +123,7 @@ class View extends Node {
     return new ViewQuery(this.ptr, { disableDistinct })
   }
 
-  async observations ({ disableDistinct } = {}) {
+  async observations ({ disableDistinct } = { disableDistinct: false }) {
```

Comment:
Other than `this.queryPrefix` being removed, how are the changes in this file related to the PR?

Reply:
Functionality-wise is only `this.queryPrefix` removed. And some chore changes.  

I understand the concern, it distracts the reviewer. (I'll keep an eye on not mixing 'chore' changes )